department-of-veterans-affairs/va.gov-cms
# VAGOV-6065: Adding user_history module and patch.

## What this does
 - Adds a user history tab to user pages for tracking user creation and editing events
 - Adds a user history list page: `/user_history/list`

## QA / Installation
 - Go to `/user_history/initialise` or visit the status report and click on the link.
 - Visit user edit page, look for history tab.
 - Visually verify that creation / editing events are being logged.
 - Go to `user_history/list` and visually verify that all users and account history appears
